Adding documentation for s3 api credentials management (#509)
cloudops/cloudmc-api-docs: source/includes/purestorage/_object_storage_api_credentials.md

The secret key should be saved immediately after generation as it will be not be retrievable afterwards.

<aside class="warning">
If you have an existing access key, this will delete and replace that key.
</aside>

<aside class="notice">
You should be saving the secret key as it will only be shown at the generation step.
</aside>

| Required                   | &nbsp;                                              				 |
|----------------------------|-------------------------------------------------------------------|
| `region` <br/>*string*     | The PureStorage Flashblade you want to create the credentials on. |

Return value:

| Attributes                 | &nbsp;                                        |
|----------------------------|-----------------------------------------------|
| `accessKeyId`<br/>_string_ | The ID for your object storage's s3 API access key.                                         |
| `secretKey`<br/>_string_   | The secret key for your object storage's s3 API access key. Should be saved.                |

> | `region`<br/>_string_      | The region the access key belongs to. Will always be 'global' for purestorage. |

Reviewer: i thought purestorage was the one that had different regions ?

Author: You caught me!

Author: https://github.com/cloudops/cloudmc-api-docs/pull/509/commits/ff8511a2ce415947f7189f250a21f63f5f3d0f1f

Author: https://github.com/cloudops/cloudmc-api-docs/pull/509/commits/fee457b324e0da8f4dfa0c3a9ce96cea0aa5868d

> | `region`<br/>_string_      | The region the access key belongs to. Will always be 'global' for purestorage. |

Author: ```suggestion
| `region`<br/>_string_      | The region the access key belongs to. Values depend on the configuration of the PureStorage plugin. |
```

> | `taskId` <br/>*string*     | The task ID related to the bucket creation.                            |

Reviewer: ```suggestion
| `taskId` <br/>*string*     | The task ID.                            |
```

| `taskStatus` <br/>*string* | The status of the operation.                                           | 

#### Delete credentials
```shell
curl --request DELETE \
  --url `https://cloudmc_endpoint/rest/services/purestorage-aaaa/emcilroy-local/apiCredentials/montreal` \
  --header 'Mc-Api-Key: your_api_key'
```

> The above command returns JSON structured like this:

```json
{
	"taskId": "0773c033-8f0c-4366-bfe3-ca544b7cb3ac",
	"taskStatus": "SUCCESS"
}
```

<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/apiCredentials/:regionName</code>

Deletes the access credentials for your purestorage environment.

Return value:

| Attributes                 | &nbsp;                                        |
|----------------------------|-----------------------------------------------|

> | `taskId` <br/>*string*     | The task ID related to the bucket creation.   |

Reviewer: ```suggestion
| `taskId` <br/>*string*     | The task ID.   |
```
